src/doc-editors/ServerGroupsEditor.vue

```diff
@@ -51,25 +51,54 @@
                 </button>
               </p>
               <p class="control">
-                <button class="button is-small has-text-danger delete-document-button"
-                        title="Delete document"
-                        data-qa="delete-document"
-                        :class="{'is-loading': isDeleteLoading}"
-                        :disabled="selectedServerGroup?.id === '__default__'"
-                        @click="deleteDoc()">
-                  <span class="icon is-small">
-                    <i class="fas fa-trash"></i>
-                  </span>
-                  <span>
-                    Delete
+                  <span class="field has-addons">
+                    <span class="control">
+                      <button class="button is-small has-text-danger delete-server-group"
+                              data-qa="delete-server-group-btn"
+                              @click="toggleDeleteServerGroupDoc()">
+                        <span class="icon is-small">
+                          <i  class="fas fa-trash"></i>
+                        </span>
+                      </button>
+                    </span>
+                    <span class="control is-expanded confirm-delete"
+                          v-if="deleteServerGroupDoc">
+                      <input class="input  is-small width-200px delete-server-group-input"
+                             data-qa="confirm-server-group-input"
+                             title="Server Group Name to Delete"
+                             ref="confirm-delete"
+                             placeholder="Write The Server Group Name to Delete"
```

Comment:
```suggestion
                             placeholder="Confirm Server Group name to delete"
```

Capitalizaing only first word and proper nouns (Server Group, Security Policy, etc.)
Changed the word `Write` to `Confirm` to better convey that it's a name confirmation rather than a delete any Server Group by name